frontend/src/components/c-title.vue

```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+div(v-html="markdownText")
+</template>
+
+<script lang="ts">
+import MarkdownIt from 'markdown-it';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  data() {
+    return {
+      markdownText: '',
+    };
+  },
+  beforeMount() {
+    try {
+      fetch('title.md').then((response) => {
+        if (!response.ok) { // file not found
+          return '';
+        }
+        return response.text();
+      }).then((text) => {
+        const md = new MarkdownIt();
+        this.markdownText = md.render(text);
+      });
+    } catch (error) {
+      // do nothing
```

Comment:
Is there any common scenario where the program control comes to the `catch` statement? If there is, we somehow need a way to tell the user that an error has occurred.

Reply:
Honestly, I have no clue if this try-catch does anything with regard to common problems, because the common problems are handled within the code itself (e.g. missing file is handled by `response.ok` check and markdown doesn't have invalid syntax). It's just to catch freaky errors in I/O that can crash the app, which I'm really not familiar with. Perhaps I could just display the error as a text in the title component, since it's likely that the generated report won't be used if there's an error in the title?

Reply:
Sure, that sounds good! You can add an error message in place of the title component.